css/wsuwp-plugin-idonate-src.less

```diff
@@ -274,7 +274,7 @@
 		padding:10px 10px;
 		background-color:#f1f1f1;
 		border-radius:2px;
-		width: calc(20% - 5px);
+		width: calc(20% - 20px);
```

Comment:
If you set this to `calc(16%)`, it works better at the "middle" resolutions (between 900-1000px). With the current calculation, the buttons get squished too close about 920-930px.